fiddler/metadata.csv

```diff
@@ -0,0 +1,3 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+fiddler.accuracy,gauge,,percent,,"The accuracy of the model",0,fiddler,accuracy,
+fiddler.prediction_drift,gauge,,percent,,"The prediction drift of the model",0,fiddler,prediction_drift,
```

Comment:
Where is this being collected?

Reply:
It should all happen inside the check.py. But check.py is not complete yet. I am trying to get the basic things work with "accuracy". Once I have that working, I am planning to start collecting the rest of the metrics.

Reply:
Got it. Do you plan on adding the other metrics in a future release or the same PR?